core/Test/Tasty/ConsoleFormat.hs

```diff
@@ -0,0 +1,24 @@
+module Test.Tasty.ConsoleFormat
+  ( ConsoleFormat(..)
+  , failFormat
+  , okFormat
+  , skippedFormat
+  , infoOkFormat
+  , infoFailFormat
+  )
+where
+
+import System.Console.ANSI
+
+data ConsoleFormat = ConsoleFormat
+  { consoleIntensity :: ConsoleIntensity
+  , colorIntensity   :: ColorIntensity
```

Comment:
These keys can be sorted differently, but I realize there may be a semantic meaning behind the order.

Reply:
I went with the original upstream codes argument order. This way the format specifications below are a pure move.

So this derivation from our canonical form was intentional to minimize diff with upstream.